sources/web/datalab/polymer/components/datalab-files/datalab-files.ts

```diff
@@ -93,6 +93,23 @@ class FilesElement extends Polymer.Element {
    * to initialize element state.
    */
   ready() {
+    // Get the last startup path. Must be called before super.ready() to beat the
+    // currentPath changed event.
+    ApiManager.getUserSettings()
```

Comment:
I think it would be useful to define ApiManager.getUserSetting(name:string, defaultValue:any) where defaultValue is optional.